onnx/onnx.proto

```diff
@@ -92,8 +103,8 @@ message GraphProto {
   optional string name = 2;   // namespace Graph
 
   // The inputs and outputs of the graph.
-  repeated string input = 3;  // namespace Tensor
-  repeated string output = 4; // namespace Tensor
+  repeated ValueInfoProto input = 3;  // namespace Value
```

Comment:
as we discussed in a chat - this is not safe. Maybe add fields with another id?

Reply:
I think we should just bump the IR version and take this change along with #53 as a single BC.

If we get this to master this week, I think we are fine.

Reply:
It certainly makes things less clean. I will assign new ids so we can look at it again.